codalab/worker/main.py

```diff
@@ -89,6 +89,13 @@ def parse_args():
         'the requested image will not be downloaded. '
         'The bundle depends on this image will fail accordingly.',
     )
+    parser.add_argument(
```

Comment:
This should not be part of this PR

Reply:
Yeah... it was from the previous PR. Sorry that I didn't start this branch out of clean workspace.. Since the previous PR is merged to master, this should be resolved by now.